lib/commons/dom/is-modal-open.js

```diff
@@ -34,26 +37,30 @@ function isModalOpen(options) {
   // in the viewport. since we aren't sure if it is or is not a modal this is
   // just our best guess of being one based on convention.
 
-  if (cache.get('isModalOpen')) {
-    return cache.get('isModalOpen');
-  }
-
-  const definiteModals = querySelectorAllFilter(
-    // TODO: es-module-_tree
-    axe._tree[0],
-    'dialog, [role=dialog], [aria-modal=true]',
-    vNode => isVisible(vNode.actualNode)
+  return cache.get(
+    'isModalOpen',
+    definiteModalsExists(definiteModalRoles) ||
+      probableModalExists(modalPercent)
   );
+}
 
-  if (definiteModals.length) {
-    cache.set('isModalOpen', true);
-    return true;
-  }
+function definiteModalsExists(roles) {
+  return Boolean(
+    querySelectorAllFilter(
+      // TODO: es-module-_tree
+      axe._tree[0],
+      roles,
+      vNode => isVisible(vNode.actualNode)
+    ).length
+  );
+}
```

Comment:
This is an antipattern to use the `Boolean` constructor. Instead return a comparison.

```suggestion
  return querySelectorAllFilter(
      // TODO: es-module-_tree
      axe._tree[0],
      roles,
      vNode => isVisible(vNode.actualNode)
    ).length > 0;
}
```